src/core/a-node.js

```diff
@@ -292,6 +292,8 @@ ANode.evtData = {};
 ANode.newMixinIdArray = [];
 ANode.oldMixinIdArray = [];
 ANode.mixinIds = {};
+ANode.DOMContentHasLoaded = false;
+document.addEventListener('DOMContentLoaded', function () { ANode.DOMContentHasLoaded = true; }, { once: true });
```

Comment:
Feels a little weird having this listener as a side-effect of the module. Another option would be to put `ANode.DOMContentHasLoaded = true;` inside `doConnectedCallback`, but that would entail repeating the assignment unnecessarily for each entity in the scene